ResearchKit/Common/ORKResult.m

```diff
@@ -1226,6 +1260,21 @@ - (id)answer {
     return nil;
 }
 
+- (NSString *)descriptionWithNumberOfPaddingSpaces:(NSUInteger)numberOfPaddingSpaces {
+    NSMutableString *description = [NSMutableString stringWithFormat:@"%@; answer:", [self descriptionPrefixWithNumberOfPaddingSpaces:numberOfPaddingSpaces]];
+    id answer = self.answer;
+    if ([answer isKindOfClass:[NSArray class]]
+        || [answer isKindOfClass:[NSDictionary class]]
+        || [answer isKindOfClass:[NSSet class]]
+        || [answer isKindOfClass:[NSOrderedSet class]]) {
+        [description appendFormat:@"\n%@\n%@>", answer, ORKPaddingWithNumberOfSpaces(numberOfPaddingSpaces)];
```

Comment:
It could look nicer if we can apply indentation to native type printings:)

<img width="577" alt="screen shot 2015-10-19 at 2 33 36 pm" src="https://cloud.githubusercontent.com/assets/11466704/10592044/719defda-766e-11e5-85fb-76d08aacd4ae.png">